test/io-tests/fixtures.sql

```diff
@@ -0,0 +1,75 @@
+\ir db_config.sql
+
+CREATE ROLE postgrest_test_anonymous;
+CREATE ROLE postgrest_test_author;
+
+GRANT postgrest_test_anonymous, postgrest_test_author TO :USER;
+
+CREATE SCHEMA v1;
+GRANT USAGE ON SCHEMA v1 TO postgrest_test_anonymous;
+
+CREATE TABLE authors_only ();
+GRANT SELECT ON authors_only TO postgrest_test_author;
+
+CREATE TABLE projects AS SELECT FROM generate_series(1,5);
```

Comment:
Ever wanted to see a table without columns but with some rows in action? Here we go :D

Reply:
That is some black magic :magic_wand: !

It's great that we now isolate the tables used in the io tests from the api tests :clap: